.goreleaser.yml

```diff
@@ -11,6 +11,13 @@ brews:
 builds:
 - ldflags:
   - -X main.version={{.Version}} -X main.commit={{.Commit}} -s -w
+  # golang.org/x/sys/windows
+  # ../go/pkg/mod/golang.org/x/sys@v0.0.0-20200803210538-64077c9b5642/windows/zsyscall_windows.go:2852:38: undefined: WSAData
+  # ../go/pkg/mod/golang.org/x/sys@v0.0.0-20200803210538-64077c9b5642/windows/zsyscall_windows.go:3125:51: undefined: Servent
+  # ../go/pkg/mod/golang.org/x/sys@v0.0.0-20200803210538-64077c9b5642/windows/zsyscall_windows.go:3139:50: undefined: Servent
```

Comment:
why do we need these?

Reply:
This is the error code that returns from the build in case anyone in the future is wondering why we skip the windows arm build.